iOS/README.md

```diff
@@ -21,14 +21,16 @@ We recommend you refer to the FAQ at https://faq.whatsapp.com/general/26000226 f
 * A sticker is an image that has a transparent background and can be sent in a
 WhatsApp chat
 * Stickers are organized into "packs". Users must explicitly add each pack to WhatsApp one-by-one, so your app should list each pack separately and each pack must have its own affordance to add it to WhatsApp (do not try to create "add all packs" operations).
+* Sticker packs must contain either static or animated stickers, never a mix of both
 * Each sticker pack must have a minimum of 3 stickers and a maximum of 30 stickers
 * Stickers must be exactly 512 x 512 pixels
 * Stickers will render on a variety of backgrounds, white, black, colored, patterned, etc. Test your sticker art on a variety of backgrounds. For this reason, we recommend you add a 8px #FFFFFF stroke to the outside of each sticker. See the sample PSD referenced at https://faq.whatsapp.com/general/26000226 for more details. 
-* Stickers must be either in PNG or the [WebP format](https://developers.google.com/speed/webp). Currently, animated WebP, animated PNG, or animated stickers of any kind are not supported. See the section [Converting to WebP](#converting-to-webp) below for information on how to create WebP files.
-* Each sticker must be less than 100KB. See the section [Tips for Reducing File Size](#tips-for-reducing-file-size) below. 
+* Stickers must be either in PNG or the [WebP format](https://developers.google.com/speed/webp). Animated stickers are supported in the WebP format only. See the section [Converting to WebP](#converting-to-webp) below for information on how to create WebP files.
+* Each static sticker must be less than 100KB and each animated sticker must be less than 500KB. See the section [Tips for Reducing File Size](#tips-for-reducing-file-size) below. 
+* Animated stickers must have frames with minimum duration of 8ms. Animation duration should not be above 10 seconds total.
```

Comment:
Please add: The first frame should say it all - WhatsApp ends the animation on the first frame after looping so please make sure your first frame is the complete image of your sticker and adjust the sequence of the stickers accordingly so users don’t see a jumpy experience going from the last to first frame. Ex. A sticker that animated “Hi!” should have the first frame show all words “Hi!”.